CHANGELOG.md

```diff
@@ -28,6 +28,7 @@ minor release, the component will be purged, so be prepared (see `Updating` sect
 - `neofs-adm morph netmap-candidates` CLI command (#1889)
 - SN network validation (is available by its announced addresses) on bootstrap by the IR (#2475)
 - `neofs-lens storage inspect` CLI command (#1336)
+- `neofs-lens` payload-only flag (#1336)
```

Comment:
does that commit relate #1336? i would write #2543. also, both commits "closes" the same issue, that is strange